examples/dmd/pore_collapse.cpp

```diff
@@ -0,0 +1,290 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: DMD on poreCollapse.
```

Comment:
Does this example depend on code beyond libROM? If so, it may not make sense to add the example in libROM.